src/app/movie/components/tags-select/tags-select.component.ts

```
@@ -54,21 +62,32 @@ export class TagsSelectComponent implements OnInit, OnDestroy {
     });
   }
 
-  addTag(id: string) {
-    const tag = this.searchResults.find(searchingTag => searchingTag._id === id);
+  createTag() {
+    this.modal
+      .showModal('Create tag', `Do you want to create a '${this.searchControl.value}' tag.`)
+      .pipe(
+        catchError(() => EMPTY),
```

I don't really like the approach of hiding errors in empty emit